code/game/gamemodes/cult/cult_items.dm

```diff
@@ -1,38 +1,67 @@
 /obj/item/weapon/melee/cultblade
 	name = "Cult Blade"
-	desc = "An arcane weapon wielded by the followers of Nar-Sie"
+	desc = "An arcane weapon wielded by the followers of a cult."
 	icon_state = "cultblade"
 	item_state = "cultblade"
 	w_class = 4
 	force = 30
 	throwforce = 10
+	sharp = 1
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 
 
 /obj/item/weapon/melee/cultblade/attack(mob/living/target as mob, mob/living/carbon/human/user as mob)
-	if(iscultist(user))
-		return ..()
-	else
-		user.Paralyse(5)
-		to_chat(user, "\red An unexplicable force powerfully repels the sword from [target]!")
-		var/organ = ((user.hand ? "l_":"r_") + "arm")
-		var/obj/item/organ/external/affecting = user.get_organ(organ)
-		if(affecting.take_damage(rand(force/2, force))) //random amount of damage between half of the blade's force and the full force of the blade.
-			user.UpdateDamageIcon()
-	return
-
-/obj/item/weapon/melee/cultblade/pickup(mob/living/user as mob)
+	if(!iscultist(user))
+		user.Weaken(5)
+		user.unEquip(src, 1)
+		user.visible_message("<span class='warning'>A powerful force shoves [user] away from [target]!</span>", \
+							 "<span class='cultlarge'>\"You shouldn't play with sharp things. You'll poke someone's eye out.\"</span>")
+		if(ishuman(user))
+			var/mob/living/carbon/human/H = user
+			H.apply_damage(rand(force/2, force), BRUTE, pick("l_arm", "r_arm"))
+		else
+			user.adjustBruteLoss(rand(force/2,force))
+		return
+	..()
+
+/obj/item/weapon/melee/cultblade/pickup(mob/living/user)
 	..()
 	if(!iscultist(user))
-		to_chat(user, "\red An overwhelming feeling of dread comes over you as you pick up the cultist's sword. It would be wise to be rid of this blade quickly.")
+		to_chat(user, "<span class='cultlarge'>\"I wouldn't advise that.\"</span>")
+		to_chat(user, "<span class='warning'>An overwhelming sense of nausea overpowers you!</span>")
 		user.Dizzy(120)
 
+/obj/item/weapon/melee/cultblade/dagger
+	name = "sacrificial dagger"
+	desc = "A strange dagger said to be used by sinister groups for \"preparing\" a corpse before sacrificing it to their dark gods."
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "render"
+	w_class = 2
+	force = 15
+	throwforce = 25
+	var/cooldown = 0
+
+/obj/item/weapon/melee/cultblade/dagger/afterattack(mob/living/target as mob, mob/living/carbon/human/user as mob)
+	..()
+	var/mob/living/carbon/human/bleeder = target
+	if(!(cooldown > world.time) && (!bleeder.stat || !(bleeder.species.flags & NO_BLOOD)))

```

Comment:
This will always proc if the victim is conscious - regardless of whether their species has blood or not.
The `||` there should probably be an `&&` instead.
If you want this to only work on non-conscious people, you can do `bleeder.stat != CONSCIOUS`